chrome/skin/style.css

```diff
@@ -151,3 +151,12 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
 #universal-search-recommendation.highlight #universal-search-recommendation-url {
   color: HighlightText;
 }
+#universal-search-recommendation.highlight #universal-search-recommendation-image {
+  background-color: #FFF;
+  box-shadow: 0;
```

Comment:
That's there to provide a bounding box for images with white or transparent backgrounds, e.g. 
<img width="1020" alt="screen shot 2016-03-17 at 2 06 17 pm" src="https://cloud.githubusercontent.com/assets/23885/13859517/716e038c-ec49-11e5-8d82-aef4a1df2fa7.png">